### PR TITLE
chore: update project documentation

### LINKED ISSUE #763: Update project README to be more relevant
The existing README is outdated, contains inaccuracies, and is overly technical. Moreover, it doesn't properly represent the current project goals and lacks essential information. This issue aims to update the document and rectify these problems.

### LINKED ISSUE #766: Update the developer guide for better setup and run instructions
Our developer documentation needs improvement. We're receiving numerous basic questions on Discord that could be answered more effectively with clearer documentation.

### PR DESCRIPTION
**Pull request by jleach**
# Summary of Changes

Updating the README contents.

# Related Issues

Fixes #763, Fixes #766 

# Pull Request Checklist

Tick all boxes below to demonstrate that you have completed the respective task. If the item does not apply to your this PR **check it anyway** to make it apparent that there's nothing to do.

- [x] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this);
- [x] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components;
- [x] Updated documentation as needed for changed code and new or modified features;
- [x] Added sufficient [tests](../__tests__/) so that overall code coverage is not reduced.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

Pro Tip 🤓

- Read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!
- Your PR will likely not be reviewed until all the above boxes are checked and all automated tests have passed.

_PR template adapted from the Python attrs project._

### CODE REVIEW

**Review by bryce-mcmath — COMMENTED** [review]
Lots of good improvements

File: DEVELOPER.md [hunk 1]
```diff
@@ -1,58 +1,183 @@
...
-## Installing dependencies
-Next, install the npm modules needed to build and test the app.
 ```sh
-# from the root of the cloned repository, run:
 yarn install
 ```
 
-## Building packages
-Some packages need to be build, transpiled befoer it can be uses from the app
+Some packages need to be build, transpiled before it can be uses from the app. Do this with the following command:
```
Comment (bryce-mcmath) [inline DEVELOPER.md]: Some packages need to be *built*, transpiled before *they* can be *used* from the app.

**Review by bryce-mcmath — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -1,154 +1,48 @@
-# Aries Mobile Agent React Native
+# Bifold README.md
 
-Aries Mobile Agent React Native is an open source Aries mobile agent app. This is a project that has been created to focus the community's efforts towards a central open source project. Various different organizations and people have expressed interest in a open source community project to help focus efforts to help prevent duplication of work between projects. Aries Bifold is also intended to help get complex or specific use-case projects started faster by cloning and having a basis of an Aries agent.
+# Summary
 
-## Component Effort
+Aries Mobile Agent React Native (Bifold) is an open-source mobile Aries client that's part of the [Hyperledger Foundation](https://www.hyperledger.org/), a global ecosystem committed to advancing enterprise-grade blockchain technologies. This project is centered around Verifiable Credentials (VC), using blockchain and best in-class cryptograpy technology to deliver trusted, secure data exchanges.
```
Comment (daidoji) [inline README.md]: >in-class _cryptography_ technology

File: DEVELOPER.md [hunk 3]
```diff
@@ -1,58 +1,183 @@
...
 
-* [Prerequisite software](#prerequisite-software)
+The following document is intended to help developers get started with the Bifold project. It includes information on how to set up your development environment, build the project, and run the app in an emulator.
 
-## Prerequisite software
+# Project Overview
 
-Before you can proceed with building and testing the BC Wallet app, you must install and configure the
-following products on your development machine:
+The Aries Mobile Agent React Native (Bifold) is a user-friendly mobile agent that is build with React Native and uses Aries Framework JavaScript (AFJ) to exchange verifiable credentials with other agents. While AFJ handles the heavy lifting of verifiable credential work, Bifold focuses on user experience and interactions with these credentials.
```
Comment (daidoji) [inline DEVELOPER.md]: >user-friendly mobile agent that is _built_ with React Native

**Review by daidoji — COMMENTED** [review]
Cool, these instructions worked for me.  I really appreciate it, these docs are a lot better.

File: DEVELOPER.md (hunk 3 above)
Comment (jleach) [reply DEVELOPER.md]: Fixed

File: README.md (hunk 2 above)
Comment (jleach) [reply README.md]: Fixed

**Review by bryce-mcmath — DISMISSED** [review]

**Review by bryce-mcmath — APPROVED** [review]

**Pull request merged**